src/native/libs/configure.cmake

```diff
@@ -1180,3 +1180,6 @@ check_c_source_compiles(
 configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/Common/pal_config.h.in
     ${CMAKE_CURRENT_BINARY_DIR}/Common/pal_config.h)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/Common/pal_config.h.in
+    ${CMAKE_CURRENT_BINARY_DIR}/Common/config.h)
```

Comment:
```suggestion
```
There is identical `configure_file` directive right before this one.

Reply:
Letting each library specify the config file to get the HAVE* constants defined in random.c. The above one creates a pal_config.h with the prefix pal.

Reply:
Instead of using the pal_config.h.in to just generate the same file with a different name, I'd rather we add a `configure.cmake` under minipal that will use a `minipal_config.h.in` (or some name like that) to generate whatever configuration is needed by minipal files. And then consumers can `include(${CLR_SRC_NATIVE_DIR}/minipal/configure.cmake)` and not have to figure out their own way to generate the configuration minipal files expect.